content/docs/command-reference/params/index.md

```diff
@@ -159,6 +159,7 @@ params.yaml  train.layers    None   9
 ```
 
 This command shows the difference in parameters between the workspace and the
-last committed version of the `params.yaml` file. In our example, there's no
-previous version, which is why all `Old` values are `None`. See `params diff` to
-learn more about the `diff` command.
+last committed version of the `params.yaml` file. In our example there's no
+previous version, which is why all `Old` values are `None`.
+
+> See `params diff` to learn more about the `diff` command.
```

Comment:
add emoji?

Reply:
If you prefer, why not. Which one though?

But it's a pretty simple note, I wouldn't usually.

Reply:
like 💡 I guess - since it's educational ... `learn more`

Reply:
Yeah could be. But actually this note is redundant now that I think about it, I just removed it 😋